repos.json

```diff
@@ -1,275 +1,410 @@
 [
     {
-        "compliant_cli": true,
-        "files": {
-            "Dockerfile": null,
-            "README.md": null,
-            "ocrd-tool.json": null,
-            "setup.py": null
-        },
-        "git": {
-            "last_commit": "Mon May 30 13:15:47 2022 +0200",
-            "latest_tag": "v2022-05-04",
-            "number_of_commits": "784",
-            "url": "https://github.com/OCR-D/ocrd_all"
+        "additional_info": {
+            "links": {
+                "Dockerfile": null,
+                "README.md": "https://github.com/ASVLeipzig/cor-asv-ann/blob/master/README.md",
+                "ocrd-tool.json": "https://github.com/ASVLeipzig/cor-asv-ann/blob/master/ocrd-tool.json",
+                "setup.py": "https://github.com/ASVLeipzig/cor-asv-ann/blob/master/setup.py"
+            }
         },
-        "name": "cor-asv-ann",
-        "ocrd_tool": "",
-        "ocrd_tool_validate": "NO ocrd-tool.json",
+        "compliant_cli": true,
+        "id": "cor-asv-ann",
+        "latest_version": "v0.1.12",
+        "ocrd_tool_json_valid": false,
         "official": true,
-        "org_plus_name": "ASVLeipzig/cor-asv-ann",
+        "project_type": "python",
         "url": "https://github.com/ASVLeipzig/cor-asv-ann"
     },
     {
-        "compliant_cli": true,
-        "files": {
-            "Dockerfile": null,
-            "README.md": null,
-            "ocrd-tool.json": null,
-            "setup.py": null
-        },
-        "git": {
-            "last_commit": "Mon May 30 13:15:47 2022 +0200",
-            "latest_tag": "v2022-05-04",
-            "number_of_commits": "784",
-            "url": "https://github.com/OCR-D/ocrd_all"
+        "additional_info": {
+            "links": {
+                "Dockerfile": null,
+                "README.md": "https://github.com/ASVLeipzig/cor-asv-fst/blob/master/README.md",
+                "ocrd-tool.json": "https://github.com/ASVLeipzig/cor-asv-fst/blob/master/ocrd-tool.json",
+                "setup.py": "https://github.com/ASVLeipzig/cor-asv-fst/blob/master/setup.py"
+            }
         },
-        "name": "cor-asv-fst",
-        "ocrd_tool": "",
-        "ocrd_tool_validate": "NO ocrd-tool.json",
+        "compliant_cli": true,
+        "id": "cor-asv-fst",
+        "latest_version": "",
```

Comment:
Latest version is here empty, shouldn't we have "unreleased_changes" in that case? I also think it may be better to have "unreleased _changes" as mandatory key. If main/master branch has no unreleased commit we should always send `"unreleased_changes":0`. This way the front-end can always rely on this key.

Reply:
Unreleased changes are not implemented yet (and not topic of this PR, see #6 ), but we can surely make it mandatory!